### ISSUE TITLE
caniuse integration 

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/respec#1238]
Would like to dynamically add caniuse integration. Basically, given a "key" or keys would like to read from:

https://github.com/Fyrd/caniuse/blob/master/features-json/

And then display the support. This could be either at the top of a spec, or per section. So:

```JS
var config = {
   caniuse: "payment-request",
}
```

And per section: 

```HTML
<section data-caniuse="payment-request">
   <h2>The foo interface</h2>
</section>
```

**Comment by saschanaz** [comment]
Recently Mozilla launched [its own data repository](https://github.com/mdn/browser-compat-data) for [MDN](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Structures/Compatibility_tables), it may be a possible alternative.

**Comment by marcoscaceres** [comment]
Oh, nice to see! We could definitely add support for that. Looks like they are still sorting out the data format.

**Comment by marcoscaceres** [comment]
Actually, this is nice checking interface compat against WebIDL in a spec. Definitely want to add support for this at some point.

**Comment by marcoscaceres** [comment]
...so, the caniuse.com integration is nice for the head of the document... and the MDN compat tables are good for when WebIDL is declared.

**Comment by marcoscaceres** [comment]
Closed via #1238
